web/client/epics/annotations.js

```diff
@@ -122,8 +122,8 @@ const validateFeatureCollection = (feature) => {
  * @param drawMethod
  * @return {boolean}
  */
-const getGeodesicProperty = (state, drawMethod = "Circle") => {
-    return drawMethod === "Circle" && get(state.annotations, "config.geodesic", false);
+const getGeodesicProperty = (state) => {
+    return get(state.annotations, "config.geodesic");
```

Comment:
I would preserve the false default value here
```suggestion
    return get(state.annotations, "config.geodesic", false);
```